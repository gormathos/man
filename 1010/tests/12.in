sol_id;user_id;task_id;lang_id;score
156678;1043;2005;1;51.06
156679;1002;2004;3;100.00
156681;1053;2002;1;73.33
156684;1044;2001;5;51.06
156688;1035;2001;1;0.00
156693;1036;2004;1;100.00
156699;1036;2005;1;0.00
156706;1099;2003;4;100.00
156714;1035;2002;4;66.67
156723;1067;2001;2;0.00
156733;1096;2005;3;82.98
156744;1028;2004;3;0.00
156756;1076;2005;1;95.74
156769;1012;2005;3;100.00
156783;1064;2004;1;81.40
156798;1072;2002;1;60.00
156814;1007;2005;1;53.19
156831;1056;2003;2;0.00
156849;1022;2004;3;0.00
156868;1034;2005;1;68.09
156888;1009;2004;3;100.00
156909;1027;2003;3;75.76
156931;1030;2004;5;86.05
156954;1019;2002;3;36.67
156978;1051;2003;3;96.97
157003;1048;2004;1;0.00
157029;1078;2003;3;54.55
157056;1073;2004;4;100.00
157084;1091;2005;5;100.00
157113;1006;2005;5;36.17
157143;1066;2005;4;0.00
157174;1084;2004;1;16.28
157206;1096;2004;1;0.00
157239;1084;2005;1;95.74
157273;1013;2001;4;61.70
157308;1057;2001;4;0.00
157344;1022;2003;3;3.03
157381;1089;2001;3;0.00
157419;1030;2005;4;70.21
157458;1066;2002;1;56.67
157498;1006;2005;3;89.36
157539;1056;2001;1;48.94
157581;1039;2005;1;23.40
157624;1027;2004;3;100.00
157668;1016;2001;5;100.00
157713;1003;2002;2;0.00
157759;1023;2001;1;100.00
157806;1082;2004;4;0.00
157854;1008;2001;2;55.32
157903;1099;2001;5;80.85
157953;1059;2001;2;53.19
158004;1083;2005;3;100.00
158056;1072;2005;3;100.00
158109;1041;2002;2;100.00
158163;1032;2004;5;65.12
158218;1004;2004;4;0.00
158274;1091;2001;5;0.00
158331;1033;2005;5;40.43
158389;1026;2003;2;0.00
158448;1040;2002;4;23.33
158508;1032;2003;4;100.00
158569;1062;2003;2;100.00
158631;1059;2005;1;2.13
158694;1003;2001;2;100.00
158758;1070;2003;1;96.97
158823;1011;2004;2;76.74
158889;1091;2004;5;100.00
158956;1075;2004;2;79.07
159024;1029;2001;3;25.53
159093;1039;2004;1;100.00
159163;1008;2002;3;70.00
159234;1051;2004;2;100.00
159306;1066;2001;5;100.00
159379;1069;2004;2;0.00
159453;1039;2004;3;100.00
159528;1079;2001;4;0.00
159604;1082;2004;2;100.00
159681;1052;2002;5;0.00
159759;1020;2005;3;82.98
159838;1097;2003;5;48.48
159918;1011;2003;1;24.24
159999;1058;2002;5;40.00
160081;1073;2004;1;0.00
160164;1085;2005;1;0.00
160248;1060;2004;1;100.00
160333;1048;2005;4;100.00
160419;1099;2002;4;23.33
160506;1025;2005;2;87.23
160594;1096;2004;2;27.91
160683;1031;2005;5;59.57
160773;1090;2005;1;72.34
160864;1068;2003;3;60.61
160956;1090;2001;1;10.64
161049;1027;2005;5;23.40
161143;1096;2004;3;0.00
161238;1090;2003;3;100.00
161334;1081;2002;4;100.00
161431;1019;2002;4;50.00
161529;1017;2005;5;0.00
161628;1061;2004;3;90.70
161728;1041;2005;1;0.00
161829;1082;2001;2;100.00
161931;1020;2005;2;0.00
162034;1057;2001;5;100.00
162138;1039;2005;4;100.00
162243;1087;2003;1;69.70
162349;1015;2002;4;0.00
162456;1044;2004;4;0.00
162564;1076;2004;1;53.49
162673;1045;2005;1;100.00
162783;1050;2005;1;89.36
162894;1061;2002;1;0.00
163006;1006;2003;5;100.00
163119;1003;2005;3;12.77
163233;1060;2001;4;0.00
163348;1084;2005;2;0.00
163464;1064;2002;1;0.00
163581;1039;2002;1;76.67
163699;1003;2005;4;100.00
163818;1095;2003;2;57.58
163938;1069;2005;5;27.66
164059;1066;2002;4;0.00
164181;1003;2005;1;59.57
164304;1030;2003;5;39.39
164428;1089;2001;4;0.00
164553;1041;2004;4;20.93
164679;1083;2003;3;100.00
164806;1075;2005;3;100.00
164934;1073;2001;2;100.00
165063;1096;2001;3;0.00
165193;1085;2001;3;0.00
165324;1049;2002;1;73.33
165456;1070;2002;5;26.67
165589;1030;2003;1;100.00
165723;1009;2003;3;42.42
165858;1028;2004;4;100.00
165994;1050;2003;4;100.00
166131;1010;2004;4;0.00
166269;1033;2001;4;100.00
166408;1026;2005;2;0.00
166548;1016;2001;5;100.00
166689;1095;2002;5;40.00
166831;1045;2004;1;41.86
166974;1018;2005;1;70.21
167118;1091;2001;1;100.00
167263;1052;2004;1;100.00
167409;1016;2002;3;0.00
167556;1004;2004;2;100.00
167704;1078;2001;2;100.00
167853;1036;2004;5;0.00
168003;1064;2001;3;0.00
168154;1050;2004;5;0.00
168306;1036;2003;2;9.09
168459;1006;2001;1;100.00
168613;1038;2001;4;0.00
168768;1012;2004;4;41.86
168924;1025;2004;2;69.77
169081;1068;2001;5;100.00
169239;1050;2005;3;100.00
169398;1088;2004;4;46.51
169558;1058;2005;2;31.91
169719;1022;2004;2;46.51
169881;1054;2003;2;100.00
170044;1097;2003;1;12.12
170208;1100;2005;2;100.00
170373;1096;2001;5;85.11
170539;1022;2001;3;100.00
170706;1007;2003;2;57.58
170874;1037;2003;2;18.18
171043;1071;2001;1;40.43
171213;1028;2005;4;78.72
171384;1033;2003;3;33.33
171556;1024;2002;1;0.00
171729;1011;2003;3;100.00
171903;1085;2003;3;100.00
172078;1020;2001;1;100.00
172254;1004;2003;3;27.27
172431;1035;2005;3;0.00
172609;1057;2003;3;60.61
172788;1001;2003;5;33.33
172968;1037;2005;4;100.00
173149;1078;2002;4;36.67
173331;1031;2005;1;59.57
173514;1075;2004;1;100.00
173698;1100;2004;3;100.00
173883;1066;2004;4;0.00
174069;1009;2001;4;100.00
174256;1059;2004;4;100.00
174444;1098;2004;4;53.49
174633;1032;2004;5;0.00
174823;1043;2004;1;100.00
175014;1083;2002;3;0.00
175206;1088;2002;3;0.00
175399;1077;2001;5;100.00
175593;1075;2003;3;15.15
175788;1071;2001;5;51.06
175984;1031;2003;4;100.00
176181;1066;2003;3;57.58
176379;1041;2005;1;40.43
176578;1036;2004;1;100.00
176778;1065;2005;3;0.00
176979;1072;2005;2;8.51
177181;1012;2001;4;0.00
177384;1064;2003;3;0.00
177588;1061;2003;2;100.00
177793;1091;2001;3;0.00
177999;1046;2003;3;100.00
178206;1063;2002;3;0.00
178414;1027;2003;5;100.00
178623;1082;2004;3;53.49
178833;1095;2005;5;0.00
179044;1039;2001;4;100.00
179256;1088;2003;5;100.00
179469;1064;2003;5;87.88
179683;1034;2004;1;0.00
179898;1088;2001;3;0.00
180114;1005;2001;4;0.00
180331;1016;2003;3;0.00
180549;1093;2002;2;0.00
180768;1064;2004;2;0.00
180988;1088;2001;2;42.55
181209;1037;2003;1;0.00
181431;1096;2001;2;8.51
181654;1058;2002;5;73.33
181878;1019;2002;2;0.00
182103;1045;2005;5;68.09
182329;1026;2004;2;48.84
182556;1098;2005;3;100.00
182784;1087;2003;5;30.30
183013;1037;2001;5;46.81
183243;1081;2002;3;83.33
183474;1078;2003;3;0.00
183706;1075;2001;3;97.87
183939;1066;2003;1;81.82
184173;1017;2005;5;0.00
184408;1053;2005;4;0.00
184644;1004;2002;2;63.33
184881;1053;2003;2;100.00
185119;1036;2002;4;100.00
185358;1059;2002;5;36.67
185598;1006;2005;3;100.00
185839;1063;2003;3;0.00
186081;1006;2003;2;0.00
186324;1054;2004;4;100.00
186568;1085;2005;4;74.47
186813;1032;2005;5;44.68
187059;1046;2003;2;84.85
187306;1098;2001;1;0.00
187554;1087;2003;5;27.27
187803;1007;2003;2;100.00
188053;1075;2001;3;34.04
188304;1009;2003;5;0.00
188556;1050;2002;3;100.00
188809;1035;2002;1;60.00
189063;1075;2005;4;0.00
189318;1050;2004;5;86.05
189574;1091;2003;5;96.97
189831;1055;2005;4;61.70
190089;1089;2001;3;78.72
190348;1021;2005;3;100.00
190608;1057;2001;4;0.00
190869;1017;2005;4;0.00
191131;1018;2005;4;100.00
191394;1005;2003;1;100.00
191658;1037;2005;1;0.00
191923;1094;2003;4;0.00
192189;1055;2005;3;0.00
192456;1027;2003;3;0.00
192724;1027;2003;2;0.00
192993;1099;2005;4;17.02
193263;1011;2004;5;100.00
193534;1004;2002;4;0.00
193806;1087;2005;5;100.00
194079;1044;2004;2;76.74
194353;1098;2004;2;0.00
194628;1018;2004;4;13.95
194904;1014;2003;5;100.00
195181;1041;2004;5;100.00
195459;1043;2004;5;0.00
195738;1001;2004;5;100.00
196018;1057;2003;1;0.00
196299;1083;2005;1;44.68
196581;1001;2001;2;0.00
196864;1031;2004;5;100.00
197148;1034;2002;2;0.00
197433;1006;2002;4;0.00
197719;1038;2002;3;26.67
198006;1076;2001;2;100.00
198294;1052;2003;5;0.00
198583;1065;2002;5;100.00
198873;1082;2002;2;53.33
199164;1078;2005;4;70.21
199456;1051;2002;3;63.33
199749;1066;2001;3;51.06
200043;1099;2003;5;0.00
200338;1060;2003;2;84.85
200634;1032;2003;3;100.00
200931;1010;2004;1;16.28
201229;1005;2002;5;100.00
201528;1018;2005;2;17.02
201828;1048;2003;1;0.00
202129;1019;2005;3;100.00
202431;1095;2004;4;0.00
202734;1038;2001;1;100.00
203038;1076;2005;4;8.51
203343;1095;2002;4;100.00
203649;1092;2005;3;68.09
203956;1004;2004;3;0.00
204264;1097;2004;3;100.00
204573;1074;2002;1;0.00
204883;1014;2004;2;0.00
205194;1098;2001;1;87.23
205506;1089;2005;2;85.11
205819;1088;2002;2;76.67
206133;1056;2003;1;12.12
206448;1019;2002;4;83.33
206764;1095;2003;4;0.00
207081;1050;2002;5;10.00
207399;1035;2003;5;48.48
207718;1006;2002;3;83.33
208038;1006;2005;2;82.98
208359;1100;2004;2;0.00
208681;1011;2001;5;100.00
209004;1096;2005;4;100.00
209328;1090;2001;5;72.34
209653;1085;2005;5;97.87
209979;1058;2004;2;0.00
210306;1013;2005;1;0.00
210634;1092;2002;3;90.00
210963;1059;2004;4;55.81
211293;1065;2001;4;8.51
211624;1046;2005;5;100.00
211956;1005;2005;2;0.00
212289;1027;2004;3;100.00
212623;1088;2001;1;100.00
212958;1037;2002;5;0.00
213294;1039;2005;1;100.00
213631;1019;2003;3;0.00
213969;1050;2005;5;0.00
214308;1054;2001;5;12.77
214648;1001;2004;5;67.44
214989;1001;2005;3;0.00
215331;1077;2001;5;100.00
215674;1099;2004;4;37.21
216018;1030;2002;1;0.00
216363;1089;2002;2;60.00
216709;1053;2004;2;100.00
217056;1039;2005;4;55.32
217404;1021;2004;4;0.00
217753;1063;2003;5;100.00
218103;1040;2002;3;16.67
218454;1034;2001;2;65.96
218806;1085;2003;4;100.00
219159;1072;2001;4;0.00
219513;1098;2003;3;0.00
219868;1078;2002;2;0.00
220224;1062;2004;5;0.00
220581;1032;2003;2;6.06
220939;1042;2003;3;100.00
221298;1051;2001;3;0.00
221658;1014;2002;2;63.33
222019;1026;2005;1;40.43
222381;1026;2001;5;38.30
222744;1079;2004;2;0.00
223108;1003;2004;1;100.00
223473;1012;2003;3;100.00
223839;1094;2003;1;57.58
224206;1033;2001;1;0.00
224574;1014;2001;1;100.00
224943;1011;2003;4;100.00
225313;1089;2002;2;0.00
225684;1037;2002;4;53.33
226056;1005;2001;3;95.74
226429;1057;2005;5;100.00
226803;1041;2002;5;66.67
227178;1041;2004;1;48.84
227554;1012;2004;2;0.00
227931;1012;2003;4;57.58
228309;1034;2003;2;0.00
228688;1037;2002;1;83.33
229068;1016;2005;1;17.02
229449;1092;2005;5;0.00
229831;1034;2001;2;100.00
230214;1074;2001;1;12.77
230598;1057;2001;5;100.00
230983;1085;2003;3;18.18
231369;1002;2002;3;0.00
231756;1062;2002;1;0.00
232144;1075;2001;3;55.32
232533;1064;2005;5;80.85
232923;1090;2004;2;72.09
233314;1021;2002;1;0.00
233706;1080;2003;3;45.45
234099;1061;2003;2;0.00
234493;1004;2004;5;67.44
234888;1045;2003;1;100.00
235284;1023;2002;5;0.00
235681;1055;2005;1;93.62
236079;1013;2001;1;100.00
236478;1010;2004;3;0.00
236878;1027;2005;2;100.00
237279;1073;2002;3;100.00
237681;1073;2004;1;0.00
238084;1009;2003;1;66.67
238488;1027;2002;4;80.00
238893;1041;2002;5;46.67
239299;1034;2003;4;100.00
239706;1048;2003;4;84.85
240114;1074;2005;1;100.00
240523;1100;2003;4;100.00
240933;1012;2001;4;100.00
241344;1047;2005;2;21.28
241756;1068;2003;2;100.00
242169;1091;2005;1;44.68
242583;1066;2005;3;0.00
242998;1048;2002;3;13.33
243414;1019;2003;1;3.03
243831;1031;2002;3;0.00
244249;1062;2005;3;100.00
244668;1002;2002;5;66.67
245088;1041;2004;2;100.00
245509;1029;2005;5;14.89
245931;1016;2003;3;42.42
246354;1009;2002;3;93.33
246778;1051;2003;1;0.00
247203;1065;2002;1;0.00
247629;1079;2002;5;0.00
248056;1091;2002;4;90.00
248484;1027;2001;4;29.79
248913;1030;2005;5;76.60
249343;1077;2003;1;0.00
249774;1084;2001;3;21.28
250206;1024;2005;3;68.09
250639;1027;2002;2;100.00
251073;1086;2005;2;6.38
251508;1022;2001;2;100.00
251944;1022;2002;4;100.00
252381;1003;2005;2;100.00
252819;1072;2005;5;0.00
253258;1060;2001;3;100.00
253698;1002;2001;1;63.83
254139;1017;2004;5;0.00
254581;1092;2005;4;10.64
255024;1074;2001;5;100.00
255468;1009;2002;1;100.00
255913;1023;2003;5;100.00
256359;1026;2005;5;0.00
256806;1026;2005;3;100.00
257254;1001;2001;5;0.00
257703;1050;2005;1;89.36
258153;1028;2001;5;14.89
258604;1012;2002;1;0.00
259056;1040;2001;1;100.00
259509;1075;2003;2;100.00
259963;1039;2001;1;80.85
260418;1086;2003;2;0.00
260874;1003;2003;3;27.27
261331;1091;2002;5;43.33
261789;1067;2004;1;23.26
262248;1071;2001;5;100.00
262708;1077;2002;1;73.33
263169;1047;2002;3;100.00
263631;1070;2003;5;0.00
264094;1031;2001;5;100.00
264558;1067;2004;2;100.00
265023;1011;2003;4;100.00
265489;1088;2002;2;0.00
265956;1028;2003;1;63.64
266424;1053;2005;2;46.81
266893;1078;2002;2;100.00
267363;1044;2002;5;100.00
267834;1042;2002;4;0.00
268306;1011;2001;2;100.00
268779;1051;2002;4;0.00
269253;1024;2004;4;97.67
269728;1079;2001;4;0.00
270204;1079;2003;5;0.00
270681;1076;2003;5;100.00
271159;1092;2001;3;100.00
271638;1046;2002;1;100.00
272118;1088;2001;5;34.04
272599;1002;2003;3;60.61
273081;1094;2002;2;63.33
273564;1071;2005;4;82.98
274048;1061;2005;3;2.13
274533;1006;2001;1;100.00
275019;1095;2001;4;0.00
275506;1096;2005;5;48.94
275994;1017;2005;5;100.00
276483;1025;2001;2;14.89
276973;1067;2003;4;100.00
277464;1020;2002;5;0.00
277956;1063;2002;5;0.00
278449;1036;2004;2;32.56
278943;1024;2003;1;6.06
279438;1055;2005;5;0.00
279934;1082;2001;4;0.00
280431;1023;2005;3;27.66
280929;1051;2002;1;0.00
281428;1003;2001;5;23.40
281928;1030;2005;3;12.77
282429;1053;2003;5;30.30
282931;1064;2002;2;20.00
283434;1032;2003;1;0.00
283938;1061;2005;4;100.00
284443;1053;2002;1;100.00
284949;1017;2005;1;100.00
285456;1040;2002;2;0.00
285964;1002;2001;2;0.00
286473;1082;2001;2;93.62
286983;1010;2005;3;2.13
287494;1022;2004;2;100.00
288006;1064;2002;3;93.33
288519;1015;2005;1;100.00
289033;1002;2001;2;0.00
289548;1014;2001;5;0.00
290064;1002;2004;2;100.00
290581;1039;2001;4;36.17
291099;1055;2001;3;87.23
291618;1030;2001;1;0.00
292138;1039;2005;1;100.00
292659;1040;2002;2;0.00
293181;1073;2004;1;34.88
293704;1086;2004;3;100.00
294228;1098;2002;1;83.33
294753;1059;2002;1;16.67
295279;1035;2003;5;0.00
295806;1004;2004;4;100.00
296334;1007;2005;1;100.00
296863;1021;2005;3;0.00
297393;1011;2005;2;0.00
297924;1035;2001;5;0.00
298456;1092;2003;3;45.45
298989;1066;2005;2;0.00
299523;1093;2005;5;100.00
300058;1083;2001;5;0.00
300594;1004;2005;4;87.23
301131;1029;2001;1;87.23
301669;1068;2005;4;17.02
302208;1054;2001;4;100.00
302748;1045;2002;2;100.00
303289;1013;2003;5;0.00
303831;1070;2001;1;95.74
304374;1056;2003;1;6.06
304918;1049;2001;1;0.00
305463;1085;2005;1;19.15
306009;1055;2001;2;70.21
306556;1099;2003;4;100.00
307104;1020;2005;2;0.00
307653;1088;2004;2;100.00
308203;1017;2004;5;100.00
308754;1022;2004;2;100.00
309306;1050;2001;5;87.23
309859;1048;2001;3;82.98
310413;1072;2003;5;15.15
310968;1025;2004;2;9.30
311524;1071;2003;1;0.00
312081;1074;2003;3;6.06
312639;1015;2003;1;100.00
313198;1070;2001;3;78.72
313758;1084;2001;5;61.70
314319;1043;2003;2;30.30
314881;1092;2004;2;23.26
315444;1005;2001;4;51.06
316008;1070;2003;5;57.58
316573;1028;2003;2;45.45
317139;1015;2003;1;100.00
317706;1095;2003;2;18.18
318274;1047;2003;5;90.91
318843;1074;2001;2;55.32
319413;1039;2004;4;0.00
319984;1030;2001;1;0.00
320556;1070;2004;5;41.86
321129;1029;2005;2;0.00
321703;1042;2002;5;56.67
322278;1062;2004;1;0.00
322854;1019;2004;1;100.00
323431;1088;2001;2;2.13
324009;1028;2004;4;100.00
324588;1064;2002;1;26.67
325168;1083;2004;2;0.00
325749;1006;2003;2;100.00
326331;1084;2004;5;100.00
326914;1031;2001;2;0.00
327498;1027;2002;1;0.00
328083;1074;2004;1;65.12
328669;1004;2001;5;100.00
329256;1041;2004;4;100.00
329844;1092;2002;1;96.67
330433;1094;2001;3;2.13
331023;1036;2003;2;0.00
331614;1013;2002;2;100.00
332206;1019;2001;2;100.00
332799;1015;2002;1;40.00
333393;1050;2003;3;0.00
333988;1015;2004;2;100.00
334584;1050;2004;1;46.51
335181;1014;2002;4;6.67
335779;1082;2004;4;34.88
336378;1073;2001;4;100.00
336978;1048;2004;2;83.72
337579;1096;2003;4;24.24
338181;1011;2001;4;4.26
338784;1050;2003;3;78.79
339388;1028;2002;1;20.00
339993;1078;2005;2;0.00
340599;1068;2004;4;51.16
341206;1058;2001;3;34.04
341814;1026;2003;4;0.00
342423;1012;2004;1;20.93
343033;1007;2004;3;0.00
343644;1099;2002;3;33.33
344256;1059;2001;5;100.00
344869;1057;2004;4;100.00
345483;1079;2002;1;96.67
346098;1097;2001;4;36.17
346714;1096;2003;5;0.00
347331;1020;2001;3;23.40
347949;1085;2001;4;0.00
348568;1010;2002;5;0.00
349188;1011;2003;1;0.00
349809;1032;2004;3;88.37
350431;1042;2005;3;100.00
351054;1073;2002;5;100.00
351678;1051;2003;2;0.00
352303;1084;2002;5;100.00
352929;1073;2002;4;0.00
353556;1086;2004;5;0.00
354184;1059;2003;5;0.00
354813;1014;2004;4;4.65
355443;1059;2005;3;8.51
356074;1015;2003;1;100.00
356706;1042;2002;5;26.67
357339;1083;2004;4;81.40
357973;1069;2003;5;81.82
358608;1019;2004;2;23.26
359244;1099;2004;5;100.00
359881;1004;2001;5;100.00
360519;1010;2003;4;0.00
361158;1026;2002;1;100.00
361798;1010;2002;1;0.00
362439;1057;2005;3;21.28
363081;1078;2002;1;0.00
363724;1007;2003;4;66.67
364368;1084;2001;1;70.21
365013;1017;2002;2;10.00
365659;1084;2005;3;0.00
366306;1087;2001;5;0.00
366954;1022;2002;5;0.00
367603;1059;2001;4;0.00
368253;1051;2001;3;100.00
368904;1039;2005;5;100.00
369556;1018;2003;3;45.45
370209;1030;2002;2;0.00
370863;1081;2001;4;0.00
371518;1066;2004;2;27.91
372174;1007;2003;2;100.00
372831;1001;2003;5;0.00
373489;1057;2005;4;0.00
374148;1022;2002;4;100.00
374808;1064;2002;3;10.00
375469;1043;2004;2;100.00
376131;1044;2005;4;40.43
376794;1094;2005;4;36.17
377458;1029;2005;1;23.40
378123;1094;2005;2;100.00
378789;1063;2002;3;36.67
379456;1058;2001;5;0.00
380124;1079;2003;4;0.00
380793;1059;2005;3;72.34
381463;1026;2003;1;54.55
382134;1100;2005;2;0.00
382806;1011;2001;1;100.00
383479;1059;2003;5;48.48
384153;1051;2003;5;100.00
384828;1063;2002;2;0.00
385504;1082;2003;1;0.00
386181;1066;2004;5;100.00
386859;1022;2003;2;0.00
387538;1087;2005;3;100.00
388218;1049;2005;3;100.00
388899;1056;2003;3;9.09
389581;1065;2005;1;61.70
390264;1038;2004;5;32.56
390948;1062;2001;3;100.00
391633;1042;2001;3;44.68
392319;1061;2001;5;80.85
393006;1052;2003;5;18.18
393694;1073;2004;3;79.07
394383;1084;2003;1;81.82
395073;1003;2001;3;19.15
395764;1047;2002;5;66.67
396456;1059;2004;4;41.86
397149;1020;2004;4;0.00
397843;1074;2004;3;100.00
398538;1062;2003;3;0.00
399234;1016;2005;2;0.00
399931;1079;2004;1;0.00
400629;1066;2003;3;100.00
401328;1057;2002;1;23.33
402028;1020;2003;5;90.91
402729;1055;2002;3;46.67
403431;1042;2004;1;81.40
404134;1058;2005;2;48.94
404838;1057;2005;4;76.60
405543;1094;2005;5;100.00
406249;1012;2003;1;27.27
406956;1025;2002;5;73.33
407664;1002;2001;1;59.57
408373;1041;2001;3;0.00
409083;1017;2004;3;0.00
409794;1044;2005;2;4.26
410506;1086;2002;4;10.00
411219;1065;2001;5;42.55
411933;1081;2004;3;0.00
412648;1034;2005;3;100.00
413364;1080;2003;2;27.27
414081;1046;2001;1;27.66
414799;1038;2005;3;8.51
415518;1003;2002;5;0.00
416238;1051;2005;4;0.00
416959;1095;2005;5;80.85
417681;1033;2003;2;0.00
418404;1009;2001;3;85.11
419128;1087;2004;2;11.63
419853;1019;2004;4;0.00
420579;1053;2005;2;23.40
421306;1068;2003;4;57.58
422034;1033;2001;1;38.30
422763;1017;2004;1;0.00
423493;1027;2001;1;0.00
424224;1038;2002;1;100.00
424956;1018;2004;1;0.00
425689;1040;2004;4;81.40
426423;1091;2003;5;0.00
427158;1098;2005;5;2.13
427894;1064;2003;5;6.06
428631;1006;2005;2;100.00
429369;1035;2005;5;87.23
430108;1016;2003;3;33.33
430848;1026;2005;5;34.04
431589;1016;2004;3;0.00
432331;1052;2004;1;100.00
433074;1062;2002;1;6.67
433818;1075;2001;2;95.74
434563;1063;2005;4;65.96
435309;1062;2001;4;51.06
436056;1043;2005;3;100.00
436804;1090;2002;5;20.00
437553;1024;2001;5;65.96
438303;1004;2004;5;100.00
439054;1047;2002;3;66.67
439806;1039;2004;3;0.00
440559;1064;2005;3;0.00
441313;1032;2002;2;0.00
442068;1080;2002;1;16.67
442824;1027;2004;1;0.00
443581;1099;2004;1;100.00
444339;1045;2005;4;0.00
445098;1006;2002;5;73.33
445858;1039;2002;1;0.00
446619;1088;2004;2;100.00
447381;1064;2002;1;3.33
448144;1003;2002;4;100.00
448908;1069;2005;2;65.96
449673;1030;2001;4;2.13
450439;1022;2004;3;13.95
451206;1090;2003;3;0.00
451974;1028;2002;4;90.00
452743;1039;2001;5;53.19
453513;1043;2002;5;93.33
454284;1067;2003;2;39.39
455056;1039;2004;4;100.00
455829;1052;2001;4;51.06
456603;1049;2001;4;93.62
457378;1040;2005;1;23.40
458154;1029;2003;2;84.85
458931;1055;2002;5;0.00
459709;1068;2004;5;0.00
460488;1029;2003;5;100.00
461268;1034;2003;1;100.00
462049;1082;2005;5;0.00
462831;1075;2004;5;62.79
463614;1032;2005;3;4.26
464398;1087;2005;5;0.00
465183;1017;2001;2;100.00
465969;1024;2002;5;0.00
466756;1048;2002;2;70.00
467544;1085;2003;5;54.55
468333;1063;2002;2;36.67
469123;1007;2001;3;80.85
469914;1089;2001;5;0.00
470706;1026;2003;3;33.33
471499;1040;2001;4;27.66
472293;1034;2003;3;0.00
473088;1045;2001;1;31.91
473884;1041;2001;3;100.00
474681;1013;2004;1;100.00
475479;1021;2003;2;100.00
476278;1028;2005;3;31.91
477078;1061;2003;5;100.00
477879;1093;2005;5;100.00
478681;1095;2005;1;76.60
479484;1036;2001;4;46.81
480288;1037;2004;5;0.00
481093;1073;2002;5;0.00
481899;1051;2002;4;66.67
482706;1048;2001;5;100.00
483514;1020;2004;2;0.00
484323;1003;2002;2;20.00
485133;1011;2002;4;100.00
485944;1004;2003;2;0.00
486756;1070;2004;5;100.00
487569;1049;2005;5;27.66
488383;1038;2004;4;0.00
489198;1034;2005;5;0.00
490014;1093;2001;5;82.98
490831;1084;2003;1;0.00
491649;1097;2004;1;0.00
492468;1021;2003;4;0.00
493288;1079;2001;1;74.47
494109;1025;2004;4;79.07
494931;1017;2002;4;10.00
495754;1097;2003;2;60.61
496578;1073;2003;4;100.00
497403;1056;2001;2;46.81
498229;1046;2003;3;93.94
499056;1073;2005;4;0.00
499884;1034;2002;1;100.00
500713;1088;2002;2;0.00
501543;1090;2004;1;100.00
502374;1054;2004;5;100.00
503206;1035;2002;5;100.00
504039;1020;2004;5;0.00
504873;1097;2004;5;100.00
505708;1013;2002;1;100.00
506544;1064;2002;3;0.00
507381;1039;2004;2;55.81
508219;1081;2004;4;34.88
509058;1081;2001;1;51.06
509898;1061;2001;4;85.11
510739;1052;2001;1;8.51
511581;1043;2003;1;24.24
512424;1032;2001;2;100.00
513268;1099;2002;5;100.00
514113;1038;2003;3;0.00
514959;1079;2005;4;12.77
515806;1073;2003;5;96.97
516654;1091;2004;3;0.00
517503;1091;2001;3;0.00
518353;1039;2005;5;0.00
519204;1040;2001;2;6.38
520056;1031;2002;2;10.00
520909;1012;2004;2;100.00
521763;1052;2005;4;80.85
522618;1075;2003;5;100.00
523474;1014;2003;1;57.58
524331;1098;2002;3;26.67
525189;1098;2004;2;100.00
526048;1001;2002;5;10.00
526908;1039;2001;5;91.49
527769;1053;2003;4;0.00
