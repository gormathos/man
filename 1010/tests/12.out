Pascal: 84 181 45 47
Python: 81 165 46 51
C++: 81 170 43 47
Java: 79 156 41 48
C#: 85 190 56 53
